### PR TITLE
refactor(addressParser): to script setup composition api

### PR DESCRIPTION
**Pull request by stephenjason89**
## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Related with #4750 
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer?target=16UcV9V6nVvPYdHz98ymUKmNLkzjCEU5sbKJMi7hxYyTHjzR&usdamount=100&donation=true)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖 Generated by Copilot at 769abca</samp>

Refactored the code for opening the shopping cart modal from the navbar. Moved the logic and configuration from `ShoppingCartModalConfig.ts` and `ShoppingCartButton.vue` to `Navbar.vue` to improve code organization and performance.

### <samp>🤖 Generated by Copilot at 769abca</samp>

> _`ShoppingCartButton`_
> _Simpler, no more `emit` -_
> _Autumn leaves falling_

### CODE REVIEW

File: components/accounts/AddressParser.vue [hunk 1]
```diff
@@ -14,24 +14,14 @@
...
-@Component({
-  components: {
-    NeoField,
-    NeoInput,
-  },
-})
-export default class AddressParser extends Vue {
-  @Prop({ type: Array, default: () => [], required: true }) value!: string[]
-
-  @Emit('input')
```
Comment (Jarsen136) [inline components/accounts/AddressParser.vue:31]: The emitting of the input event is missing in the refactored code.

Comment (stephenjason89) [reply components/accounts/AddressParser.vue:31]: @Jarsen136 thanks for pointing it out. It's now included.

**Comment by yangwao** [comment]
this is highly sensitive stuff, recommend test this one well 
cc @prury

**Review by Jarsen136 — APPROVED** [review]

**Comment by prury** [comment]
was able to input addresses properly ✔️ 
number of total addresses was correct ✔️ 
simple mint transferred the NFTs to addresses correctly ✔️ 
https://kusama.subscan.io/tx/0xaae93b4c05c6b9883e3804f08eb93402e60d202b90cb4c6137a1170400301eb8
https://kusama.subscan.io/tx/0xfd1a8ec25f3622e7cf852bd897cc93906766ad3f977cb6516f3258224c32fd70

**Review by roiLeo — APPROVED** [review]
✅ code lgtm, keep it up

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 5.03 USD/DOT ~ 9.94 $DOT
🧗 [16UcV9V6nVvPYdHz98ymUKmNLkzjCEU5sbKJMi7hxYyTHjzR](https://polkadot.subscan.io/account/16UcV9V6nVvPYdHz98ymUKmNLkzjCEU5sbKJMi7hxYyTHjzR)
🔗 [0xf78c81721f516c292968082cc783aa5f7edbecf3ef6d6983e337222657bce808](https://polkadot.subscan.io/extrinsic/0xf78c81721f516c292968082cc783aa5f7edbecf3ef6d6983e337222657bce808)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**